### PR TITLE
HydrusNetwork.HydrusNetwork version 505b

### PR DESCRIPTION
**Pull request by Ryonez**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/88229)

### CODE REVIEW

**Comment by Ryonez** [comment]
@microsoft-github-policy-service agree

File: manifests/h/HydrusNetwork/HydrusNetwork/505b/HydrusNetwork.HydrusNetwork.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: HydrusNetwork.HydrusNetwork
+PackageVersion: 505b
```
Comment (ItzLevvie) [inline manifests/h/HydrusNetwork/HydrusNetwork/505b/HydrusNetwork.HydrusNetwork.installer.yaml:5]:
```suggestion
PackageVersion: v505b
```
This must match Control Panel.

File: manifests/h/HydrusNetwork/HydrusNetwork/505b/HydrusNetwork.HydrusNetwork.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: HydrusNetwork.HydrusNetwork
+PackageVersion: 505b
```
Comment (ItzLevvie) [inline manifests/h/HydrusNetwork/HydrusNetwork/505b/HydrusNetwork.HydrusNetwork.locale.en-US.yaml:5]:
```suggestion
PackageVersion: v505b
```
This must match Control Panel.

File: manifests/h/HydrusNetwork/HydrusNetwork/505b/HydrusNetwork.HydrusNetwork.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: HydrusNetwork.HydrusNetwork
+PackageVersion: 505b
```
Comment (ItzLevvie) [inline manifests/h/HydrusNetwork/HydrusNetwork/505b/HydrusNetwork.HydrusNetwork.yaml:5]:
```suggestion
PackageVersion: v505b
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by Ryonez** [comment]
My apologies Levvie, I'll keep that mind with any further work I do. Changes have been made.

**Comment by Ryonez** [comment]
Screw it, I can't see an easy way to change the path...
Using wincreate to make a new on here: https://github.com/microsoft/winget-pkgs/pull/88253

**Comment by mdanish-kh** [comment]
> Screw it, I can't see an easy way to change the path...

fyi: You can do this easily here in GitHub by pressing the <kbd>.</kbd> (dot) key on your keyboard. That'll open up the changes brought in by this PR in VSCode Online where you can easily rename the folder from the explorer menu 
image

Once you've made your changes, you can stage and commit them from the Source Control menu on the left (the changes are automatically pushed).

**Comment by Ryonez** [comment]
Thank you, that is really neat @mdanish-kh. Though I do note it warns me it can't find the source branch for the pull.

**Comment by mdanish-kh** [comment]
> Thank you, that is really neat @mdanish-kh. Though I do note it warns me it can't find the source branch for the pull.

That's probably because you deleted the branch. The tip was just an fyi for future PRs.

**Comment by Ryonez** [comment]
Heh, forgot about that.

**Pull request closed without merging**